controller/rq_goals.ml

```diff
@@ -8,11 +8,12 @@
 (* Replace by ppx when we can print goals properly in the client *)
 let mk_message (_loc, lvl, msg) =
   let hdr =
-    if lvl = 4 then Pp.str "[info ] "
-    else if lvl = 5 then Pp.str "[debug] "
+    if lvl = 2 then Pp.str "[warning] "
+    else if lvl = 4 then Pp.str "[info   ] "
+    else if lvl = 5 then Pp.str "[debug  ] "
```

Comment:
```suggestion
    else if lvl = 4 then Pp.str "[ info  ] "
    else if lvl = 5 then Pp.str "[ debug ] "
```

Reply:
That does align poorly with `[warning]` , doesn' t it?

Reply:
Let's send the level to the client instead.